src/mapeo-manager.js

```diff
@@ -143,4 +158,56 @@ export class MapeoManager {
 
     return project
   }
+
+  /**
+   * @returns {Promise<Array<ProjectInfo & Pick<ProjectValue, 'name'>>>}
+   */
+  async listProjects() {
+    const allProjectsResult = this.#db
+      .select({
+        projectId: projectKeysTable.projectId,
+        keysCipher: projectKeysTable.keysCipher,
+      })
+      .from(projectKeysTable)
+      .all()
```

Comment:
was thinking about a potentially premature optimization where we use the active project instances from memory (`this.#activeProjects`) and then only query for the inactive projects, not sure the extra code is worth it considering the _n_ seems like it's pretty small and probably doesn't make this existing query an expensive one.